Gemfile

```diff
@@ -5,3 +5,6 @@ gemspec
 
 # Include coveralls for CI coverage reports
 gem 'coveralls', require: false
+
+gem 'parliament-grom-decorators', path: '../parliament-grom-decorators'
```

Comment:
If these are being used for testing we will need to either add them as development dependencies or look at how they are needed within the app.